From fb82d060ff5754ff0ea0ff44e1fb39fa5b01caa5 Mon Sep 17 00:00:00 2001
From: huzhen <george.hu@spacemit.com>
Date: Tue, 26 Mar 2024 14:40:41 +0800
Subject: [PATCH 0521/1202] eth: change the range of dma to range1

  As the dma of gmac can access 32 bit range, so it can access
  phy addr 4G~6G(dram 2G~4G).

Change-Id: Ic0512ad05b87381abc7741a4faff40b7b8a348ad
---
 arch/riscv/boot/dts/spacemit/k1-x.dtsi | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/riscv/boot/dts/spacemit/k1-x.dtsi b/arch/riscv/boot/dts/spacemit/k1-x.dtsi
index 7e10f8d0e5d5..fe7cb8562aef 100644
--- a/arch/riscv/boot/dts/spacemit/k1-x.dtsi
+++ b/arch/riscv/boot/dts/spacemit/k1-x.dtsi
@@ -1596,7 +1596,7 @@ eth0: ethernet@cac80000 {
 			ptp-support;
 			ptp-clk-rate = <10000000>;
 			lpm-qos = <7>;
-			interconnects = <&dram_range0>;
+			interconnects = <&dram_range1>;
 			interconnect-names = "dma-mem";
 			status = "disabled";
 		};
@@ -1618,7 +1618,7 @@ eth1: ethernet@cac81000 {
 			ptp-support;
 			ptp-clk-rate = <10000000>;
 			lpm-qos = <7>;
-			interconnects = <&dram_range0>;
+			interconnects = <&dram_range1>;
 			interconnect-names = "dma-mem";
 			status = "disabled";
 		};
-- 
2.47.0

